### PR TITLE
Update clones with info for bitcoin

### PR DESCRIPTION
**Pull request by sgliner-ledger**

### CODE REVIEW

File: pages/docs/device-app/develop/code/clone.mdx [hunk 1]
```diff
@@ -33,11 +33,16 @@ Depending on your derivation path, the process will vary.
 
 #### New Bitcoin clone
 
-For any other clone type, use the [main Device App process](./quickstart/).
+Instead of forking the boilerplate app you should fork [app-bitcoin-clone-boilerplate](https://github.com/LedgerHQ/app-bitcoin-clone-boilerplate).
 
 #### Update an existing clone
```
Comment (tdejoigny-ledger) [inline pages/docs/device-app/develop/code/clone.mdx:39]: not sure that we need two have 2 sections here

File: pages/docs/device-app/develop/code/clone.mdx [hunk 2]
```diff
@@ -33,11 +33,16 @@ Depending on your derivation path, the process will vary.
```
Comment (tdejoigny-ledger) [inline pages/docs/device-app/develop/code/clone.mdx:33]: now we have also bitcoin clone, so it seems that using "cards" for Ethereum clones is not really a good idea. WDYT @cfranceschi-ledger

File: pages/docs/device-app/develop/code/clone.mdx [hunk 3]
```diff
@@ -33,11 +33,16 @@ Depending on your derivation path, the process will vary.
...
+Instead of forking the boilerplate app you should fork [app-bitcoin-clone-boilerplate](https://github.com/LedgerHQ/app-bitcoin-clone-boilerplate).
 
 #### Update an existing clone
 
-If your Bitcoin clone was included in the Bitcoin Deice App and you need to update it, please start a [new Device App process](../../introduction/).
+If your Bitcoin clone was included in the Bitcoin Device App and you need to update it, you should
+create a new application by forking [app-bitcoin-clone-boilerplate](https://github.com/LedgerHQ/app-bitcoin-clone-boilerplate).
+
+You can then add your icon, customize the makefile, update the README, add your tests and you should be good to go !
+
```
Comment (tdejoigny-ledger) [inline pages/docs/device-app/develop/code/clone.mdx:44]: CI shall be green (requirements are almost the same as for a standard app)

File: pages/docs/device-app/develop/code/clone.mdx (hunk 2 above)
Comment (cfranceschi-ledger) [reply pages/docs/device-app/develop/code/clone.mdx:33]: Yes, you are right @tdejoigny-ledger. I will merge this PR and rework the organisation of the page on another branch.

**Review by cfranceschi-ledger — APPROVED** [review]

**Pull request merged**